CHANGELOG.md

```diff
@@ -1,5 +1,51 @@
 # Changelog Identity And Consent Management
 
+## Table of Contents
+
+- [v0.2.0-rc](#v020-rc)
+- [v0.1.0 - Initial version](#v010---initial-version)
+
+# v0.2.0-rc
```

Comment:
According to  https://wiki.camaraproject.org/display/CAM/Meta-release+Process#MetareleaseProcess-Versionandreleasenumbering the version number should follow the template `x.y.z-rc.n`

```suggestion
# v0.2.0-rc.1
```
Then the links and subsequent occurrences should be modified accordingly.

Reply:
@sebdewet @AxelNennker, Jesús is on holiday and as I am not codeowner I cannot send a commit. Can you do it for me?, I'm fine with this correction. The change is a replace all _0-rc_ to _0-rc.1_, except in line 5 where has to be _0-rc1_ to make link work. Thanks!